index.js

```diff
@@ -55,7 +55,7 @@ function FirebaseServer(port, name, data) {
 	this.Firebase = firebaseCopy();
 	this.name = name || 'mock.firebase.server';
 	this.Firebase.goOffline();
-	this.baseRef = new this.Firebase('ws://fakeserver.firebaseio.com');
+	this.baseRef = new this.Firebase('ws://fakeserver.firebaseio.test');
```

Comment:
This needs to make it in regardless.

Ending in `.com` causes error messages to be logged to the console. Running the new `watch` target leaves the process open long enough that a LOT more of them get logged, and it's really annoying.